test-commons/src/main/java/no/unit/nva/publication/PublicationGenerator.java

```diff
@@ -234,14 +234,14 @@ private static FileSet sampleFileSet() {
 
     private static License randomLicense() {
         return new License.Builder()
-            .withIdentifier(randomString())
+            .withIdentifier(FAKER.lorem().word())
             .withLink(randomUri())
             .withLabels(randomMap())
             .build();
     }
 
     private static Map<String, String> randomMap() {
-        return Map.of(randomString(), randomString());
+        return Map.of("no", randomString());
```

Comment:
A string literal is not very random. If you need a random language and you do not have time, create a randomLanguage() that returns a "no" and then we can expand that method later

Reply:
+1